src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/TaskList.tsx

```diff
@@ -38,12 +38,13 @@ const TaskList = ({ project, defaultClosed }: TaskListProps) => {
   if (!user) return <LoadingIndicator />;
 
   const createTaskPermissions =
-    !(user.role === 'GUEST' && !project.team?.members.map((user) => user.userId).includes(user.userId)) &&
-    !(
-      user.role === 'MEMBER' &&
-      (project.projectLead?.userId !== user.userId || project.projectManager?.userId !== user.userId) &&
-      !(project.team?.leader.userId === user.userId)
-    );
+    user &&
```

Comment:
Don't think you need this since we have the loading indicator thing above

Reply:
u right